cul.js

```diff
@@ -136,7 +141,8 @@ var Cul = function (options) {
 
     this.write = function send(data, callback) {
         //console.log('->', data)
-        serialPort.write(data + '\r\n', callback);
+        serialPort.write(data + '\r\n');
+        serialPort.drain(callback);
```

Comment:
I've tested the example and it also works. But I definitely need the serialPort.drain function otherwise the callback isn't called in my case.
  
```
serialPort.write(data + '\r\n', function(){
    serialPort.drain(callback);
});
```